compliant-sets/many-operations/overlay.yaml

```
@@ -0,0 +1,23 @@
+overlay: 1.0.0
+info:
+  title: Structured Overlay
+  version: 1.0.0
+actions:
+- target: $.paths['/pets'].get.summary
+  remove: true
+- target: $.paths.*.get
```

```suggestion
- $.paths.*.get.tags
```

Hey @lornajane, I have a question about this and the related output.yaml:

The spec says:

> If the target selects an object node, the value of this field MUST be an object with the properties and values to merge with the selected node. If the target selects an array, the value of this field MUST be an entry to append to the array. 

The related part of the output expects "wildcard-done" to be added to the list of tags. But the target ($.paths.*.get) does not select an array. So should this be `$.paths.*.get.tags` instead?

---

Sorry, I should probably just ask somewhere else about how to read the spec. The spec says "merge", which I understand like `Object.assign(a, b) `  in JavaScript, where if properties defined in a will be overwritten by the same properties defined b. Array values would just be overwritten and not concatenanted.

Is my understanding wrong?